src/snowflake/snowpark/modin/plugin/extensions/io_overrides.py

```diff
@@ -146,7 +145,14 @@ def json_normalize(
 ) -> pd.DataFrame:  # noqa: PR01, RT01, D200
     # TODO(https://github.com/modin-project/modin/issues/7104):
     # modin needs to remove defaults to pandas at API layer
-    pass  # pragma: no cover
+    _pd_json_normalize_signature = {
+        val.name
+        for val in inspect.signature(native_pd.json_normalize).parameters.values()
+    }
+    _, _, _, f_locals = inspect.getargvalues(inspect.currentframe())
```

Comment:
Why use inspect instead of passing the arguments directly?

Reply:
I believe this should help us stay consistent with upstream Modin.